src/ShortcutGuide.tsx

```diff
@@ -22,282 +27,63 @@ const ShortcutGuide: React.FC = () => {
 
 	return (
 		<>
-			<div className="a11y_modal" data-testid="a11y_modal">
+			<div className="shortcut-guide__background" data-testid="shortcut-guide">
```

Comment:
Like mentioned above, define these classes in a object so that we can re-use them in tests, other files that query these, etc.